Source/Urho3D/Graphics/Shader.h

```diff
@@ -72,9 +74,9 @@ class URHO3D_API Shader : public Resource
     /// Source code adapted for pixel shader.
     ea::string psSourceCode_;
     /// Vertex shader variations.
-    ea::unordered_map<StringHash, SharedPtr<ShaderVariation> > vsVariations_;
+    ea::unordered_map<unsigned, SharedPtr<ShaderVariation> > vsVariations_;
```

Comment:
Why change key to `unsigned`?

Reply:
Because it's `StringHash` no more, obviously.
Meaning that there's no meaningful string that would result in such hash.